app/assets/javascripts/components/ops/tenant-component.js

```diff
@@ -0,0 +1,94 @@
+ManageIQ.angular.app.component('tenantComponent', {
+  bindings: {
+    recordId: '=?',
```

Comment:
`@?` does not seem to work.

When I load the Tenant Editor, the `Name` field is prepopulated with the `tenants` string.
<img width="1419" alt="screen shot 2017-07-21 at 12 19 05 pm" src="https://user-images.githubusercontent.com/1538216/28479318-c14aaa1e-6e10-11e7-8c41-9d6db0b75ba7.png">